caravel/controllers/forms.py

```diff
@@ -33,11 +33,11 @@ class ListingForm(flask_wtf.Form):
     
 
 class NewListingForm(ListingForm):
-    principal = PrincipalField("Seller",
-        validators=[DataRequired(), OnlyValid()])
+    principal = PrincipalField("Seller", validators=[DataRequired()])
```

Comment:
Hmm... I had thought that there might be Hospital staff with @uchicago.edu email addresses that cannot sign into Google Apps, but on second thought, that might actually be true. (As in, email ends with @uchicago.edu iff they can sign in with Google Apps.) I've emailed someone in ITS to confirm.

(Originally, I removed the check so that hospital staff could enter their UChicago email.)